### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bodom0015**
Fixes https://github.com/cheese-hub/cheesehub/issues/7

### CODE REVIEW

**Review by BenGalewsky — APPROVED** [review]
Yep - this is a good starting place. We can add more once we know what development even looks like

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+We love pull requests from everyone. By participating in this project, you
+agree to abide by the thoughtbot [code of conduct].
```
Comment (craig-willis) [inline CONTRIBUTING.md]: thoughtbot?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+We love pull requests from everyone. By participating in this project, you
```
Comment (craig-willis) [inline CONTRIBUTING.md]: Pull requests and issues?

**Review by craig-willis — APPROVED** [review]
Good starting place.  A few minor points, if you want to consider them.

**Pull request merged**